README.md

```diff
@@ -292,6 +292,14 @@ TCP может отсылать данные вне основного пото
 * Linux: `make`
 * Windows: `make windows CC=x86_64-w64-mingw32-gcc`
 
+------
+### Docker
+
+```sh
+docker build ./ -t byedpi:latest
+docker run -p 1080:1080 -ti byedpi:latest ciadpi --disorder 1 --auto=torst --tlsrec 1+s 
```

Comment:
Add debug logs for docker:
`--disorder 1 --auto=torst --tlsrec 1+s --debug 1`

Reply:
> Add debug logs

This is just an example! You need to make your parameters yourself!